test/JDBC/input/alter/alter-mvu-test-vu-prepare.sql

```diff
@@ -0,0 +1,57 @@
+create database alter_func_db;
+go
+
+use alter_func_db
+go
+
+CREATE TABLE alter_func_users ([Id] int, [firstname] varchar(50), [lastname] varchar(50), [email] varchar(50));
+CREATE TABLE alter_func_orders ([Id] int, [userid] int, [productid] int, [quantity] int, [orderdate] Date);
+
+INSERT INTO alter_func_users VALUES (1, 'j', 'o', 'testemail'), (2, 'e', 'l', 'testemail2');
+INSERT INTO alter_func_orders VALUES (1, 1, 1, 5, '2023-06-25'), (2, 1, 1, 6, '2023-06-25');
+GO
+
+create function alter_func_mvu_f1() returns int begin return 2 end
+go
+
+alter function alter_func_mvu_f1(@param1 int) returns int begin return @param1 end
+go
+
+create function alter_func_mvu_f4() returns TABLE as return (select * from alter_func_users)
+go
+
+alter function alter_func_mvu_f4() returns TABLE as return (select * from alter_func_orders)
```

Comment:
are we allowed to ALTER FUNCTION if there are dependent objects on the function itself? for ex if it's a column default of a table. 

if it is possible, can you add a test for that? to make sure that the dependencies are properly updated.

Reply:
+ please add a test for this as well as showing that pg_depend entries are no longer orphaned

Reply:
Updated the pg_depend check + added the check for babel metadata